recipes/lora_finetune_single_device.py

```diff
@@ -328,6 +328,11 @@ def _setup_profiler(
 
         log.info(f" Profiler config after instantiation: {profiler_cfg}")
 
+        self.profiler_wait_steps = profiler_cfg["wait_steps"]
```

Comment:
This seems to be throwing an error in our recipe tests, can you just quickly inspect `profiler_cfg` after running e.g.

```
pytest tests/recipes/test_lora_finetune_single_device.py -m integration_test -k 'test_loss'
```

I thought these fields would be defined based on [this](https://github.com/pytorch/torchtune/blob/2f8ed7a34457eaddce59a58e8c37dd51d6ba65f8/torchtune/utils/_profiler.py#L370), but seems like something weird is happening here.

Reply:
Ah, it's because profiling isn't enabled for tests, so it returns here https://github.com/pytorch/torchtune/blob/2f8ed7a34457eaddce59a58e8c37dd51d6ba65f8/torchtune/utils/_profiler.py#L262